openfish-webapp/src/api/index.ts

```diff
@@ -0,0 +1,6 @@
+export type Result<T> = {
```

Comment:
I feel like the file could be result.ts to match the others convention of being named with the type that they contain, also not to be confused with ts relating to index.html.

Reply:
I chose `index.ts` so that my imports look like `import type { Result } from "./api"` . Typescript chops off the `index` part in the import. So here I am saying that `Result` belongs to `api` because it is a shared between all the different API responses.